packages/circuits/lib/fp.circom

```diff
@@ -41,6 +41,7 @@ template FpMul(n, k) {
     component q_range_check[k];
     signal r[k];
     component r_range_check[k];
+    component r_p_range_check = BigLessThan(n,k);
```

Comment:
`r_p_range_check` can be renamed to `r_p_lt_check` (or similar) as we usually use "range check" for checking bit length range.